### ISSUE TITLE
Machine Learning related books on Fortran?

### ISSUE THREAD

**Issue by lime-n** [Beliavsky/Fortran-related-books#1]
Hey, just as the title mentions - Do you know of any good books on Machine Learning for Fortran?

I have read your reading list and notice this specific field is missing and it's becoming increasingly popular in the field of scientific computing. 

Regards,
Lime

**Comment by Beliavsky** [comment]
I don't know of Fortran books on ML, although there a few Fortran books listed on statistics. Some Fortran packages for ML are [listed](http://fortranwiki.org/fortran/show/Libraries) at the Fortran Wiki:

FortLearner - linear regression, logistic regression, stochastic gradient descent, decision trees, ensemble trees, boosting trees, and dimension reduction.

neural-fortran is a parallel neural net microframework. (OS: MIT)

NN_MOD - Fortran 2008 Module/Class for training neural network regressors

autofit is a Fortran code for fitting permutationally invariant polynomials (PIPs). (OS: Apache-2.0 License)
